### DISCUSSION TITLE
simpson

### DISCUSSION THREAD

**Discussion by nuts65** [pipeeeeees/waze-voicepack-links#24] [Ideas]
anywhere to find a homer simpson voice link?

**Comment by pipeeeeees** [comment]
Hey! As far as I know, Waze has not made an official pack in collaboration with the Simpsons. Maybe sometime in the future they will.

The only other option would be a user-made one. There is one on the readme in French by someone by the name Superflame. I am unaware of any in English if that is what you are after. 

I will leave this issue open for visibility.

**Comment by nuts65** [comment]
appreciate the prompt reply.

**Comment by pipeeeeees** [comment]
Converting to an Idea Discussion rather than an issue
